Makefile

```diff
@@ -222,8 +222,7 @@ $(GCI):
 ##@ End-to-end (e2e) Testing
 
 e2e-test: ## Run e2e tests
-	chainsaw test --test-dir test/e2e/scenario --config test/e2e-config/.chainsaw.yaml
-	chainsaw test --test-dir test/e2e/workload-scan --config test/e2e-config/.chainsaw.yaml
+	chainsaw test --test-dir test/e2e/scenario --test-dir test/e2e/workload-scan --config test/e2e-config/.chainsaw.yaml
```

Comment:
Could do the chainsaw stuff in a separate PR

Reply:
It is quite related, so I think it's fine to have it in the same PR.